### PR TITLE
Add support for video tracking analytics

### PR DESCRIPTION
**Pull request by ccnixon**
Adds integration between our video spec and parse.ly's [video tracking api](https://www.parse.ly/help/integration/video/).

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -97,6 +97,31 @@ Parsely.prototype.page = function(page) {
  */
 
 Parsely.prototype.track = function(track) {
+  var event = track.event();
+  var videoEvents = {
+    'Video Content Started'     : 'trackPlay',
```
Comment (hankim813) [inline lib/index.js]: we should ideally just handle these events separately via:

```
Parsely.prototype.videoContentStarted = function(track) { ..};
Parsely.prototype.videoPlaybackPaused = function(track) { ..};
Parsely.prototype.videoPlaybackInterrupted = function(track) { ..};
```

File: lib/index.js [hunk 2]
```diff
@@ -97,6 +97,31 @@ Parsely.prototype.page = function(page) {
...
+    'Video Playback Interrupted': 'reset'
+  };
+
+  if (videoEvents.hasOwnProperty(event) && window.PARSELY.video) {
+    var vidId = track.proxy('properties.assetId');
+    var urlOverride = track.proxy('context.page.url');
+
+    // https://www.parse.ly/help/integration/video/#video-metadata
+    // https://paper.dropbox.com/doc/Segment-Video-Spec-jdrVhQdGo9aUTQ2kMsbnx
+    var metadata = reject({
```
Comment (hankim813) [inline lib/index.js]: since we are using `reject`, are any of these properties required?

File: lib/index.js [hunk 3]
```diff
@@ -97,6 +97,31 @@ Parsely.prototype.page = function(page) {
...
+      title: track.proxy('properties.title'),
+      pub_date_tmsp: + new Date(track.proxy('properties.airdate')),
+      image_url: track.proxy('properties.imageUrl'),
+      section: track.proxy('properties.genre'),
+      authors: [track.proxy('properties.publisher')],
+      tags: track.proxy('properties.tags')
```
Comment (hankim813) [inline lib/index.js]: properties.tags is not spec'd. the video content events has spec'd `properties.keywords` which is an array of tags so I think we should use that.

File: lib/index.js (hunk 2 above)
Comment (ccnixon) [inline lib/index.js]: Yes, some are required but don't we generally just let their api handle errors?

**Comment by ccnixon** [comment]
@hankim813 - made some changes per your feedback.

**Pull request merged**